Jenkinsfile_CNP

```diff
@@ -48,24 +48,29 @@ def vaultOverrides = [
     'spreview': 'saat'
 ]
 
-// vars needed for functional tests
+// Env variables to switch pipeline logic and wiring per type of build
+env.DATA_STORE_DEVELOP_PR       = "pr-601" // This doesn't change frequently, but when it does, only change this value.
+env.DEFINITION_STORE_DEVELOP_PR = "pr-575" // This doesn't change frequently, but when it does, only change this value.
+env.ACTIVITY_API_DEVELOP_PR     = "PR-211" // This doesn't change frequently, but when it does, only change this value.
+env.PRs_TO_USE_AAT              = "PR-217" // Set this value to a PR number, or add it as a comma-separated value, if it's to follow CI/CD.
```

Comment:
Maybe worth adding to the comment that this one importantly needs to always be uppercase, to work with L#84?

Reply:
made case-insensitive.